src/coordinator/index.js

```diff
@@ -161,6 +176,28 @@ class Coordinator {
     // it could errantly remove underscores from service IDs.
     return this._request.get(`${this._baseUrl}/users/${userId}/permissions`);
   }
+
+  /**
+   * Gets information about all contxt applications
+   *
+   * API Endpoint: '/applications'
+   * Method: GET
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtApplication[]} Information about all contxt applications
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator
+   *   .getAllApplications()
+   *   .then((apps) => console.log(apps))
+   *   .catch((err) => console.log(err));
+   */
+  getAllApplications() {
```

Comment:
Would you mind moving this function above `getUserPermissionsMap`? We've been trying to keep the functions inside these modules alphabetized. Thanks, @smstromb!

Reply:
Will do 👍